### PR TITLE
ADD new test to check attribute order in notifications also in no-value modes

### PR DESCRIPTION
**Pull request by fgalan**
A complement of #1875 , increasing test cases coverage (in particular, attribute orderin in notifications for keyValues and default NGSIv2 notification formats).

@crbrox 
@kzangeli

### CODE REVIEW

File: test/functionalTest/cases/1875_NGSIv2_notification_formats/NGSIv2_notification_formats_order_in_values.test [hunk 1]
```diff
@@ -77,7 +77,6 @@ payload='
   "expires": "2050-04-05T14:00:00.00Z"
 }'
 orionCurl --url /v2/subscriptions --payload "$payload" --json
-sub5=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
```
Comment (fgalan) [inline test/functionalTest/cases/1875_NGSIv2_notification_formats/NGSIv2_notification_formats_order_in_values.test]: It seems to be a leftover, sub5 is never used...

Comment (kzangeli) [inline test/functionalTest/cases/1875_NGSIv2_notification_formats/NGSIv2_notification_formats_order_in_values.test]: NTC

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**